image/Dockerfile.engine-dm

```diff
@@ -0,0 +1,79 @@
+# Common builder
+ARG GO_IMAGE
+FROM ${GO_IMAGE} as golang
+
+FROM centos:7 as builder
+ENV GOPATH=/go
+ENV PATH $PATH:/usr/local/go/bin:$GOPATH/bin
+ENV AUTO_GOPATH 1
+COPY --from=golang /usr/local/go /usr/local/go
+COPY hack/dockerfile/install/tini.installer /
+COPY hack/dockerfile/install/proxy.installer /
+RUN yum install -y \
+    bash \
+    ca-certificates \
+    cmake \
+    gcc \
+    git \
+    glibc-static \
```

Comment:
also, this one is not needed since we're not building statically. Does not harm either, so removing it is just for clarity.

Reply:
sorry I overlooked it earlier

also maybe `cmake` is not needed either, although I'm not so sure

Reply:
We use cmake later on in the `tini` compilation so I think it'd need to stay.

Reply:
right

Reply:
glibc-static, I guess, is also needed for tini. As a followup, we can move installing these two under `FROM builder as init-builder`